pom.xml

```diff
@@ -124,6 +124,49 @@
           </execution>
         </executions>
       </plugin>
+      <!-- Spotless plugin: keeps the code formatted following the google-java-styleguide -->
+      <plugin>
+      <groupId>com.diffplug.spotless</groupId>
+      <artifactId>spotless-maven-plugin</artifactId>
+      <version>2.40.0</version>
+      <executions>
```

Comment:
The indentation here for `<executions>` and `<configuration>` seems wrong; it should probably be indented by 2 additional spaces.

And `<formats>` should be indented by 4 additional spaces.

Reply:
Oh yes, good catch!